spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -251,11 +251,7 @@ private void deployStream(StreamDefinition stream, Map<String, String> streamDep
 		}
 	}
 
-	/**
-	 * Return a copy of a given app definition where short form parameters have been expanded to their long form
-	 * (amongst the whitelisted supported properties of the app) if applicable.
-	 */
-	/*default*/ StreamAppDefinition qualifyParameters(StreamAppDefinition original, Resource resource) {
+	private Map<String, String> qualifyParameters(Map<String, String> appDeploymentProperties, Resource resource) {
```

Comment:
That is the name of the variable in the code that calls out to this... it's a bit confusing but it's intention is to capture the fact that it is the groups of properties *from* `stream deploy` that apply to a particular *app*. What do you suggest as an alternative?